internal/interaction/transaction.go

```diff
@@ -179,7 +194,18 @@ func (s *serviceInteractor) UpdateTransaction(ctx context.Context, tran *entitie
 		curTran.Deletion.Status = curTran.TransactionStatus
 		curTran.TransactionStatus = entities.TransactionStatusDeleted
 
-		return s.store.DeleteTransaction(ctx, curTran)
+		if err := s.store.DeleteTransaction(ctx, curTran); err != nil {
+			return err
+		}
+
+		// inform the attendee service that a transaction was deleted
+		if err := s.attendeeClient.PaymentsChanged(ctx, uint(tran.DebitorID)); err != nil {
```

Comment:
safer to do this only for payments (could theoretically create an infinite loop for dues)